src/num/multind.c

```diff
@@ -1525,7 +1531,7 @@ bool md_compare2(unsigned int D, const long dims[D], const long str1[D], const v
 
 		bool eq2 = (0 == memcmp(ptrs[0], ptrs[1], size2));
 
-		#pragma omp critical
+		#pragma omp atomic
```

Comment:
This change is also not related to Windows compatibility, just rather a minor enhancement.